packages/code-du-travail-nlp/api/sem_search.py

```diff
@@ -27,17 +22,19 @@ def __init__(self, stops_path: str):
 
         with open(stops_path, "r") as f:
             stops = f.read().splitlines()
+            stops.extend(["travail",
+                          "travailler",
+                          "travaille"])
```

Comment:
why not in the data file ?

Reply:
because the data file is currently used by both the sem search and the suggester and removing "travail" from suggestions is a bad idea I guess 😄

Reply:
suggester is gone, stopwords in no more shared ! we can update the stop_words list (and retrieve only the stopwords file)